packages/constants/src/payload/payload.types.ts

```diff
@@ -16,20 +18,45 @@ export interface WebsiteRequestPayload {
 }
 
 export interface PaymentRequestPayload {
-  // 	The amount of currency to deliver (in drops)
-  amount: string;
+  // The amount to deliver, in one of the following formats:
+  // - A string representing the number of XRP to deliver, in drops.
+  // - An object where 'value' is a string representing the number of the token to deliver.
+  amount: Amount;
   // The unique address of the account receiving the payment
   destination: string;
-  // The token that can be used
-  currency?: string;
-  // The issuer of the token
-  issuer?: string;
-  // The memo to attach to the transaction
-  memo?: string;
+  // The memos to attach to the transaction
+  // Each attribute of each memo must be hex encoded
+  memos?: Memo[];
   // The destination tag to attach to the transaction
-  destinationTag?: string;
+  destinationTag?: number;
```

Comment:
You changed it to string to have something closer to XRPL.JS?

Reply:
From string to number yes
https://xrpl.org/payment.html#payment

https://js.xrpl.org/interfaces/Payment.html#DestinationTag